### PR TITLE
wip: simplify direct

### PR DESCRIPTION
**Pull request by szmarczak**
#### this PR

- [x] simplify direct

#### [wip: simplify forward](https://github.com/apify/proxy-chain/pull/153)

- [x] remove babel
- [x] simplify forward handler
- [x] added https support
  - [ ] test
- [x] require node.js 14
- [ ] skipped 2 tests (will re-enable once proxy-chain is fully rewritten)

#### [wip: simplify custom response](https://github.com/apify/proxy-chain/pull/154)

- [x] simplify custom response
- [x] use shared eslint config

#### TODO (those will be separate PRs)

- [ ] simplify chain
- [ ] simplify server
- [ ] simplify tools
- [ ] use typescript

### CODE REVIEW

File: src/direct.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const net = require('net');
+
+/**
+ *
+ * @param {http.ClientRequest} request
```
Comment (B4nan) [inline src/direct.js:5]:
```suggestion
 * @param {http.ClientRequest} request
```

File: src/direct.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    socket.on('error', () => {
+        source.destroy();
+    });
+
+    source.on('error', () => {
+        socket.destroy();
+    });
+};
+
+module.exports = { direct };
```
Comment (B4nan) [inline src/direct.js:40]: probably don't matter, but from what i remember, the re-assignment to `module.exports` was discouraged in favour of assigning separate properties

```suggestion
module.exports.direct = direct;
```

**Review by B4nan — APPROVED** [review]
lgtm, just two nitpicks

File: src/direct.js (hunk 2 above)
Comment (szmarczak) [reply src/direct.js]: Correct, thanks!

**Pull request merged**